README.md

```diff
@@ -1,2 +1,24 @@
-# Module-1-First-Project
-This the Module 1, Day 2 Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Day 2 Project
+> Description the project.
+This is a day 2 project
+## Built With
+
+- Major languages: English
+- Frameworks: Nill
+- Technologies used: HTML, CSS
+
+## Getting Started
+
+One can check the inclusion of CSS into HTML
```

Comment:
[OPTIONAL]
It is always a good idea to add the steps to take in order to have the project cloned locally. Please add that here